r/R/dplyr-functions.R

```diff
@@ -391,7 +470,7 @@ nse_funcs$str_split <- function(string, pattern, n = Inf, simplify = FALSE) {
     string,
     options = list(
       pattern =
-      opts$pattern,
+        opts$pattern,
```

Comment:
I know this is just linting but IDK why opts$pattern is on its own line instead of next to = above it.